orchestration/dags/data_gcp_dbt/models/export/backend/exp_backend__marketing_pro_live_show_email_churned_40_days_ago.sql

```diff
@@ -0,0 +1,20 @@
+{{
+    config(
+        **custom_incremental_config(
+        incremental_strategy = "insert_overwrite",
+        partition_by = {"field": "execution_date", "data_type": "date", "granularity" : "day"},
+        on_schema_change = "sync_all_columns",
+    )
+) }}
+
+select
+    DATE('{{ ds() }}') as execution_date,
+    venue_id,
+    venue_booking_email
+from {{ ref('mrt_global__venue') }} venue
+join {{ ref('mrt_global__offerer') }} offerer ON venue.venue_managing_offerer_id=offerer.offerer_id
+    and DATE_DIFF(CURRENT_DATE, offerer.last_bookable_offer_date, day) >= 40
```

Comment:
```suggestion
    and DATE_DIFF(date('{{ ds() }}'), offerer.last_bookable_offer_date, day) >= 40
```
à la place du CURRENT_DATE c'est mieux de mettre la date du run du dag :)